core/client/fs/src/main/java/alluxio/client/file/cache/store/LocalPageStore.java

```diff
@@ -62,19 +74,26 @@ public LocalPageStore(LocalPageStoreOptions options) {
     mFileBuckets = options.getFileBuckets();
     // normalize the path to deal with trailing slash
     Path rootDir = Paths.get(mRoot);
-    // pattern encoding root_path/page_size(ulong)/bucket(uint)/file_id(str)/page_idx(ulong)/
-    mPagePattern = Pattern.compile(
-        String.format("%s/%d/(\\d+)/([^/]+)/(\\d+)", Pattern.quote(rootDir.toString()), mPageSize));
+    // pattern encoding for each page
+    // root_path/page_size(ulong)/bucket(uint)/file_id(str)/mtime(long)/page_idx(ulong)/
+    mPagePattern = Pattern.compile(String
+        .format("%s/%d/(\\d+)/([^/]+)/(\\d+)/(\\d+)", Pattern.quote(rootDir.toString()),
+            mPageSize));
+    // pattern encoding for the directory of each file
```

Comment:
you mean parent dir, right? I only see this path down to bucket.
better to also have something like the following in java doc like mPagePattern

```
// root_path/page_size(ulong)/bucket(unit)
```